help/view-links-on-mobile.md

```diff
@@ -0,0 +1,23 @@
+# View links on mobile
+
+By default, the mobile app will open web links by switching to your device's default browser. However, if you want to open web links while using the Zulip
+mobile app, you can enable a convenient feature to seamlessly view a website
+without leaving the app.
+
```

Comment:
I think we can stick with our general approach of not describing defaults unless there's an important reason to do so, so that we don't have to worry about the documentation if defaults change.

Reply:
Yeah, that makes sense. How about:

You can configure the Zulip mobile app to seamlessly view websites without
leaving the app. This feature allows you to open web links directly in the app
instead of switching to your device's default browser.